Exiled.API/Features/Map.cs

```diff
@@ -457,6 +459,20 @@ public static Pickup GetRandomPickup(ItemType type = ItemType.None)
             return pickups[Math.Max(0, random.Next(pickups.Count - 1))];
         }
 
+        /// <summary>
+        /// Given the name of a camera, returns its <see cref="CameraType"/>, or <see langword="null"/> if the camera does not exist.
+        /// </summary>
+        /// <param name="cameraName">The name of the camera.</param>
+        /// <returns>The <see cref="CameraType"/>, or <see langword="null"/> if it does not exist.</returns>
+        public static CameraType? GetCameraType(string cameraName)
+        {
+            Camera079 cam = GetCameraByName(cameraName);
+            if (!cam)
+                return null;
```

Comment:
Don't make it Null
You can just return string.Empty

Reply:
This doesn't compile, `string.Empty` can't be casted to a `CameraType`. Although I can make it return `CameraType.Unknown` instead of `null`.

Reply:
Yes sorry i am dumb